cmd/podman/play_kube.go

```diff
@@ -182,7 +182,7 @@ func playKubeYAMLCmd(c *cliconfig.KubePlayValues) error {
 	}
 
 	for _, container := range podYAML.Spec.Containers {
-		newImage, err := runtime.ImageRuntime().New(ctx, container.Image, c.SignaturePolicy, c.Authfile, writer, &dockerRegistryOptions, image2.SigningOptions{}, false, nil)
+		newImage, err := runtime.ImageRuntime().New(ctx, container.Image, c.SignaturePolicy, c.Authfile, writer, &dockerRegistryOptions, image2.SigningOptions{}, false, nil, false)
```

Comment:
i think it is high time these things be made into a struct?  wdyt?

Reply:
@mheon says the same thing hahah

Reply:
@baude @mheon I can refactor into a struct but which elements make sense to include? Just all of them?

Reply:
Leave the ctx out of the struct, put the rest in, IMO

Reply:
Way ahead of you :)

Reply:
I like leaving forcePull and update out, it isn't quite about the image config but rather about the way we pull the image. WDYT @mheon